src/script.js

```diff
@@ -151,9 +151,9 @@ function decompilePushOnly (script) {
 
   return chunks.map(function (op) {
     if (Buffer.isBuffer(op)) return op
-    if (op === OPS.OP_0) return new Buffer(0)
+    if (op === OPS.OP_0) return op
 
-    return scriptNumber.encode(op - OP_INT_BASE)
+    return op
   })
```

Comment:
They have the effect of pushing a vch onto the stack like every other push, but we don't rely on this behavior much (example, I've never once checked the check mulltisig dummy value), but it's the semantics we need.